demo/index.js

```diff
@@ -0,0 +1,108 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
We should avoid committing compiled files to version control. Suggest adding this file to `.gitignore`